cou/steps/execute.py

```diff
@@ -54,29 +55,27 @@ def normal(text: str) -> str:
         :return: text formatted.
         :rtype: str
         """
-        return Style.RESET_ALL + Fore.RED + text + Style.RESET_ALL
+        return Style.RESET_ALL + text + Style.RESET_ALL
 
-    return (
-        normal(parameter + " (")
-        + bold("c")
-        + normal(")ontinue/(")
-        + bold("a")
-        + normal(")bort/(")
-        + bold("s")
-        + normal(")kip:")
-    )
+    return normal(parameter + "Continue(") + bold("y") + normal("/") + bold("N") + normal("):")
```

Comment:
I think that we should use `Continue (y/n)` instead of capital `N`, because capital looks like default value, what is not true.